exonum/src/blockchain/schema.rs

```diff
@@ -20,6 +20,20 @@ use helpers::Height;
 use super::{Block, BlockProof, Blockchain};
 use super::config::StoredConfiguration;
 
+// TODO: Replace by `const` when/if `contact!` accepts it.
+macro_rules! core_prefix { () => ( "core." ) }
+
+const TRANSACTIONS: &str = concat!(core_prefix!(), "transactions");
```

Comment:
Looks very ugly. Can you try to rewrite macro?

```rust
core_index_name!(TRANSACTIONS_STATUS => "transactions_status");
```

Or like this!

```rust
define_indexes! {
      const NAME = "core";
      index TRANSACTIONS_STATUS => "transactions_status";
}
```

Or any another option.